src/screens/DiscoveryScreenV2/index.js

```diff
@@ -298,13 +340,7 @@ const DiscoveryScreenV2 = ({route}) => {
           hideBackIcon
         />
       )}
-      <ScrollView
-        style={styles.fragmentContainer}
-        contentContainerStyle={styles.fragmentContentContainer}
-        keyboardShouldPersistTaps="handled"
-        onMomentumScrollBegin={handleScroll}>
-        {renderFragment()}
-      </ScrollView>
+      {renderFragment()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ScrollView` component was removed and replaced with a call to `renderFragment()`. If the `renderFragment()` function does not return a scrollable component, this could potentially lead to UI issues where content is not fully visible or accessible. Please ensure that the components returned by `renderFragment()` are properly scrollable if needed.
```diff
- {renderFragment()}
+ <ScrollView
+   style={styles.fragmentContainer}
+   contentContainerStyle={styles.fragmentContentContainer}
+   keyboardShouldPersistTaps="handled"
+   onMomentumScrollBegin={handleScroll}>
+   {renderFragment()}
+ </ScrollView>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->